js/app.jsx

```diff
@@ -517,6 +537,8 @@ var Job = React.createClass({
       ['State', state],
       ['Input', inputs],
       ['Output', cleanJobPath(job.conf.output)],
+      ['Pipe', pipe],
+      ['Step', step],
```

Comment:
we add these lines to pipes, but the pipes are aggregated into steps. Note in this case the steps has everything that is in pipes.

You could merge them both, sort and dedup might be a fine way to go, or you can just do steps. Not sure how defensive you want to be.